class NotFoundError(Exception):
    pass


class UpdateError(Exception):
    pass
